### DISCUSSION TITLE
What causes errors with the External TX module?

### DISCUSSION THREAD

**Discussion by EhAye** [EdgeTX/edgetx#2605] [Q&A]
Running ELRS HM ES24TX (not the pro version) and a TX16S. 
ELRS packet rate is set to 100(Full) and Telemetry Std(1:32)
TX Baud rate set to 5.25M

In the External RF screen, there is always 1 error when i open it up the first time after boot. And there's at least 2 or 3 errors every minute. They come in bursts. Sometimes i'll get 10-20 errors all in a row

I've tried shaking the radio a bit to see if there's a ribbon cable loose or something. I tried re-seating the external module. I double-checked the jumper locations and inspected the board for any debris... No difference.
i do get errors when its sitting still on the desk.

Error rate seems reduced a bit when setting TX Baud Rate to 3.75M.

I like to fly my quads at ELRS packet rate of 500 and 5.25m Baud. I don't notice any issues with this, in spite of the errors. I do hear 'telemetry lost/recovered' occasionally but it doesn't appear to have any effect on my control.

How far can i reduce the baud rate before it harms my elrs link?

**Comment by pfeerick** [comment]
The lower the packet rate, the lower the baud you actually need, and you can go up to 500hz at @ 400k with no trouble. All you have to do is look at the [Good/Bad indicator](https://www.expresslrs.org/3.0/quick-start/transmitters/lua-howto/#check-if-your-tx-is-connected) at the top of the ELRS Lua to see if the speed is ok, and recent versions of the ELRS Lua will also tell you the serial baud rate is too slow. If you are getting the rate you set, then the serial baud rate is fast enough. I believe extremely high baud rates like 5.25M are prone to incessant telemetry lost/found messages, and unless you are using the fast F1000 type modes (and probably even then - 1.87M seems to be enough for it) not necessary. 

The error indicator is an indicator of how many packets have been lost between the ELRS module and the transmitter - the internal telemetry. This can be because the module didn't respond properly in a packet exchange, which is usually the case during powerup, hence the omnipresent 1. This is expected behaviour. You will potentially get more errors if you switch models and the RF module gets power cycled. You'll also get errors when you change baud rates. If there is a mismatch between the baud-rate of the transmitter and the baud-rate of the ELRS module, you will get a small number of errors during a session. If it's a severe mismatch, you'll be able to see that number ticking like crazy. The odd packet loss is not an issue - the module/handset link is a lot faster than the OTA link, so it is highly unlikely you'd notice it. And increasingly so if you push the board rate up - i.e. the errors will increase, but so will the rate of transmission and retry. But
